spec/rollout_spec.rb

```diff
@@ -0,0 +1,6 @@
+RSpec.describe "Rollout" do
+  it "has two custom groups for BEIS users and PO users" do
+    expect(ROLLOUT.groups).to include(:beis_users)
```

Comment:
Do you think it'd be worth it/possible to test that a specific user belongs to a group here? e.g. create a BEIS user and check that they're in the `:beis_users`?